### DISCUSSION TITLE
Can you make disassemble code like  this?

### DISCUSSION THREAD

**Discussion by CatCatCatDog** [pwndbg/pwndbg#2131] [Ideas]
<img width="1459" alt="not good" src="https://github.com/pwndbg/pwndbg/assets/23658192/621730a7-961a-4c3c-b804-598be971f14d">

Above disassemble is complicated.

But, Below disassemble is much better.

Can you make like this on pwndbg?

<img width="1467" alt="good" src="https://github.com/pwndbg/pwndbg/assets/23658192/57e8d4a0-3f60-4607-be6f-5891e65770ac">

**Comment by disconnect3d** [comment]
Hi, 

First of all, the first screenshot doesnt seem to show disassembly context window but registers. I am not quite sure why do you see multiple instructions and symbols there -- it should be one at maximum, but its still not good that it covers all the screen, so I'd say it is to be improved. Please create a GH issue if this behavior still persist on the latest dev branch (maybe it was fixed in the meantime but I doubt so).
